modules/core-user-password.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="core-user-password_{context}"]
+= Creating a user and password for node access
+
+You can use a machine config to create a user and password for a node that allows you to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC). This is useful in the event you cannot access the node through SSH or by using the `oc debug node` command. This user account and password enables you to get into the node in order to troubleshoot a problem or view logs.  
+
+The Machine Config Operator (MCO) creates a user named `core`, assigns the password, and injects the password into the `/etc/shadow` file. The MCO does not examine the password hash. Thus, the MCO cannot report if there is a problem with the password.
+
+[NOTE]
+====
+* The password works only from the serial console. It does not work with SSH.
+
+* If you have a machine config that include a systemd unit that sets a password or adds a file to the `/etc/shadow` directory, it takes  precedence over the password hash.
+====
+
+You can change the password by editing the machine config you used to create the user and password. Also, you can remove the password by deleting the machine config. Deleting the machine config does not remove the user account from the  `/etc/shadow` file. 
```

Comment:
if the mc is used to create new user/pwd (not user `core`), when it is deleted, the user won't be deleted? @sergiordlr can you confirm this?

Reply:
@sergiordlr Thank you. I completely missed this. I was not familiar with the core user.  So, I searched the internet for Linux core user and RHEL core user. But not on RHCOS, so I assumed we were creating it.